sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/_evaluate.py

```diff
@@ -208,7 +209,11 @@ def _evaluate(
 
         def _get_instance_table():
```

Comment:
Nit pick: why is this an inline function?

Reply:
It started as an inline way to convert data to jsonl that is needed in artifacts produced. Long term plan is to break it down into different eval scenarios like qa, chat.